service/src/main/java/org/apache/griffin/core/metric/MetricStoreImpl.java

```diff
@@ -56,7 +56,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 @Component
 public class MetricStoreImpl implements MetricStore {
 
-    private static final String INDEX = "griffin";
+    @Value("${index}")
```

Comment:
1. The property name "elasticsearch.index" fits better in my opinion.
2. Could you please add default value for property?
`@Value("${elasticsearch.index:griffin}")`